laws/shared/src/main/scala/algebra/laws/RunSerializationLaws.scala

```diff
@@ -0,0 +1,9 @@
+package algebra.laws
+
+sealed trait SerializationCheck
```

Comment:
what if this is `SerializationCheck[M]` so we can default to doing `Run...` but override with `Skip...`.